lib/client.py

```diff
@@ -12,17 +12,19 @@
 import sys
 import time
 
-import shared
-from DBThriftAPI import CheckerReport
-from DBThriftAPI.ttypes import SuppressBugData
 from thrift import Thrift
 from thrift.protocol import TBinaryProtocol
 from thrift.transport import TSocket
 from thrift.transport import TTransport
 
-from codechecker_lib import suppress_file_handler
-from codechecker_lib.logger import LoggerFactory
-from storage_server import report_server
+from api import shared
```

Comment:
The same here for the `api`, can you think of a more descriptive name? It is a bit too general.